### ISSUE TITLE
Won't Load

### ISSUE THREAD

**Issue by timtim17** [timtim17/stacking-cards#1]
Won't load in chrome. Can't open DevTools.

Taking out online refs. doesn't work either.

Opening in IE loads fine without JS, w/ JS causes white screen.

**Comment by timtim17** [comment]
Page shows up with f17bb4907bec25bbfec67052de00874e2d4ecafe and ef69068316433eb194b43b65cd16f2e1c006dd3f, then goes to white. Also removed easteregg with 2b7a1754c6da5cda2c00822436252b17cb31f8ef, will need to readd. ~~Will try removing error message.~~

**Comment by timtim17** [comment]
Problem seems to be in the `regenerate()` function...

Anyone got anything?

``` javascript
// ...
function regenerate() {
        stack_container.innerHTML = "";

        if (radio_stack_three.checked) {
            stack_one = new Stack(generateRandomCard());
            stack_two = new Stack(generateRandomCard());
            stack_three = new Stack(generateRandomCard());

            stack_one = stack_one.createStack();
            stack_two = stack_two.createStack();
            stack_three = stack_three.createStack();
        } else if (radio_stack_two.checked) {
            stack_one = new Stack(generateRandomCard());
            stack_two = new Stack(generateRandomCard());
            stack_three = null;

            stack_one = stack_one.createStack();
            stack_two = stack_two.createStack();
        } else if (radio_stack_one.checked) {
            stack_one = new Stack(generateRandomCard());
            stack_two = null;
            stack_three = null;

            stack_one = stack_one.createStack();
        }

        carddeck.createStockDeck();

        // Log the carddeck to make sure it is correct.
        console.log(carddeck);

        heldCard = generateRandomCard();
        stack_held.setAttribute('src', heldCard.img);

    }
// ...
```

Found in f97e679040d0632a2e9ee0f23eb581de48adeae1

**Comment by timtim17** [comment]
Fixed in c1043d8df79e5d31e5c2827092babd29df4c5a34. Merged in #2 and 60fb9db5552e0ea1ca966af08262f008ac773a83.
